src/frameworks/next/index.ts

```diff
@@ -74,8 +77,8 @@ import {
   ROUTES_MANIFEST,
   APP_PATH_ROUTES_MANIFEST,
   APP_PATHS_MANIFEST,
-  ESBUILD_VERSION,
   SERVER_REFERENCE_MANIFEST,
+  ESBUILD_VERSION,
```

Comment:
Can we bump this dependency, change to a semver range, and warn if the existing global esbuild does not match? I understand esbuild is 0.x right now which makes this all riskier

Reply:
`^0.19.2` should be fine IMO, they could break any point though

Reply:
I have updated the PR to use `^0.19.2` and show a warning if the installed version does not match.